### PR TITLE
added brazilian portuguese translation for exchange and teamwork PSA

### PR DESCRIPTION
**Pull request by botoserelepe**
@lh1008 here is the translation of "exchange" text for review. :)

### CODE REVIEW

**Comment by lh1008** [comment]
Thank you @botoserelepe, we now we need to review this. Pinging @netrik182 or @ratin89us.

**Comment by netrik182** [comment]
Hi @lh1008. I'm doing this one.

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Conheça a segurança da sua corretora
+
+Corretoras podem ser uma maneira conveniente de vender e comprar Monero, mas nem todas elas foram criadas da mesma forma.
+
+### Aqui estão algumas dicas para você ter em mente quando se trata de guardar seus XMR de forma segura nas corretoras.
+
+1. Faça a sua própria pesquisa antes de abrir conta em qualquer corretora. Tente responder essas perguntas:
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
1. Faça a sua própria pesquisa antes de abrir uma conta em qualquer corretora. Tente responder essas perguntas:
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- a. Existem avaliações positivas de outras pessoas que a utilizam? Essas avaliações são confiáveis?
+
+- b. Essa corretora já foi atacada? O combate ao ataque foi apropriado?
+
+- c. A quanto tempo a corretora está operando?
+
+- d. Os criadores/administradores são conhecidos ou são indivíduos ou entidades de reputação?
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
- d. Os criadores/administradores são conhecidos ou são indivíduos ou entidades com boa reputação?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- e. É possível de manter sua privacidade, não fornecendo nenhuma informação pessoal?
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
- e. É possível manter sua privacidade, não fornecendo nenhuma informação pessoal?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- f. O website utiliza precauções de segurança como captcha ou autenticação de dois fatores (2FA)? Ele possui certificados de segurança legítimos e começa com https://?
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
- f. O website utiliza precauções de segurança como Captcha ou autenticação de dois fatores (2FA)? Ele possui certificados de segurança legítimos e começa com https://?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- g. Se você planeja declarar impostos, a corretora permite que você faça o download de comprovantes de transações de compra/venda?
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
- g. Se você planeja declarar impostos, a corretora permite que você faça o download de comprovantes de transações de compra e venda?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+2. As corretoras criam carteiras próprias delas para você. Você dificilmente terá acesso a semente mnemônica da carteira da corretora. Se a corretora for atacada ou representar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é apenas guardar XMR em corretoras se você planeja utilizá-los em breve. Caso contrário, transfira para uma carteira da qual você tenha controle.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
2. As corretoras criam carteiras próprias delas para você. Você dificilmente terá acesso à semente mnemônica da carteira da corretora. Se a corretora for atacada ou tentar dar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é apenas guardar XMR em corretoras se você planeja utilizá-los em breve. Caso contrário, transfira para uma carteira na qual você tenha controle.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+3. Se você quiser trocar uma quantidade significativa de XMR por meio de uma corretora, considere fazer um teste primeiro, com uma transação de valor mais baixo. Esse valor deve ser baixo o suficiente para que você não se sinta mal no caso de perdê-lo.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
3. Se você quiser trocar uma quantidade significativa de XMR por meio de uma corretora, considere fazer um teste primeiro, com uma transação de valor mais baixo. Esse valor deve ser baixo o suficiente para que você não se sinta mal caso venha a perdê-lo.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+4. Confira sempre, em qualquer transação de XMR, os endereços das carteiras que vão enviar/receber Monero para ter certeza de que estão corretos. Mesmo que você tenha copiado e colado os endereços, confira se você copiou o endereço correto, pois existem ataques que modificam endereços de criptomoedas quando estes são copiados e colados. Se o seu aparelho estiver infectado por esse vírus, uma simples dupla verificação para confirmar o endereço copiado será o suficiente para proteger seus XMR.
+
+5. Com o passar do tempo, as corretoras descentralizadas serão mais comuns. Como o Monero é uma criptomoeda descentralizada, essas corretoras estão de acordo com o caráter do Monero. Considere utilizá-las, mas como sempre, faça sua pesquisa primeiro!
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
5. Com o passar do tempo, as corretoras descentralizadas serão mais comuns. Como o Monero é uma criptomoeda descentralizada, essas corretoras estão de acordo com o espírito do Monero. Considere utilizá-las, mas como sempre, faça sua pesquisa primeiro!
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# Conheça a segurança da sua corretora
+
+Corretoras podem ser uma maneira conveniente de vender e comprar Monero, mas nem todas elas foram criadas da mesma forma.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
As corretoras podem ser uma maneira conveniente de vender e comprar Monero, mas nem todas elas foram criadas da mesma forma.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+### Aqui estão algumas dicas para você ter em mente quando se trata de guardar seus XMR de forma segura nas corretoras.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
### Confira a seguir algumas dicas para você guardar seus XMR com segurança em corretoras.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Faça a sua própria pesquisa antes de abrir uma conta em qualquer corretora. Tente responder essas perguntas:
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
1. Sempre faça a sua própria pesquisa antes de abrir uma conta em qualquer corretora. Ou seja, informe-se por conta própria, não confie apenas em recomendações de outras pessoas. Ao analisar a segurança de uma corretora, tente responder as seguintes perguntas:
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+- a. Existem avaliações positivas de outras pessoas que a utilizam? Essas avaliações são confiáveis?
+
+- b. Essa corretora já foi atacada? O combate ao ataque foi apropriado?
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
- b. Essa corretora já sofreu algum ataque hacker? As medidas para combater o ataque foram apropriadas?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+
+- c. A quanto tempo a corretora está operando?
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
- c. Há quanto tempo a corretora está em operação?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+
+- d. Os criadores/administradores são conhecidos ou são indivíduos ou entidades com boa reputação?
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
- d. Os seus fundadores e/ou administradores são conhecidos na comunidade? Eles são indivíduos ou entidades com boa reputação?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+
+- e. É possível manter sua privacidade, não fornecendo nenhuma informação pessoal?
+
+- f. O website utiliza precauções de segurança como Captcha ou autenticação de dois fatores (2FA)? Ele possui certificados de segurança legítimos e começa com https://?
+
+- g. Se você planeja declarar impostos, a corretora permite que você faça o download de comprovantes de transações de compra e venda?
+
+2. As corretoras criam carteiras próprias delas para você. Você dificilmente terá acesso a semente mnemônica da carteira da corretora. Se a corretora for atacada ou tentar dar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é apenas guardar XMR em corretoras se você planeja utilizá-los em breve. Caso contrário, transfira para uma carteira da qual você tenha controle.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md]:
```suggestion
2. Embora as corretoras criem carteiras de Monero para você, você não terá controle total sobre os fundos dessas carteiras, pois as corretoras não costumam fornecem acesso à semente mnemônica que gerou a carteira. Se a corretora sofrer um ataque hacker ou tentar dar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é guardar em corretoras apenas os XMR que você planeja usar em breve. Se você quer guardar alguns XMR a longo prazo, o recomendado é que você os transfira para uma carteira que você tenha total controle (uma carteira que você tenha a semente mnemônica).
```

File: monero-outreach-docs/translations/pt-br/PSA/files/teamwork/teamwork_pt-br.md [hunk 16]
```diff
@@ -0,0 +1,13 @@
...
+
+Se você é um desenvolvedor, você pode se unir a algumas centenas de outros desenvolvedores que contribuíram com o código Monero, via GitHub.
+
+O primeiro passo a se fazer quando se deseja começar um projeto com participação comunitária é entrar em contato com o grupo da Comunidade Monero, cujo foco é organizar todos os demais grupos de trabalho. Eles serão capazes de informar-lhe se a sua ideia já foi desenvolvida por alguém e com quais pessoas você deve trabalhar para aprimorá-la. No caso de a sua ideia ser completamente nova, eles disponibilizarão alguns recursos para auxiliá-lo, permitindo que sua ideia se torne realidade.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/teamwork/teamwork_pt-br.md]:
```suggestion
O primeiro passo a se fazer quando se deseja começar um projeto com participação comunitária é entrar em contato com o grupo da Comunidade Monero, cujo foco é organizar todos os demais grupos de trabalho. Eles serão capazes de informar-lhe se a sua ideia já foi desenvolvida por alguém e com quais pessoas você deve trabalhar para aprimorá-la. Caso a sua ideia seja completamente nova, eles disponibilizarão alguns recursos para auxiliá-lo, permitindo que sua ideia se torne realidade.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/teamwork/teamwork_pt-br.md [hunk 17]
```diff
@@ -0,0 +1,13 @@
...
+
+Se você é um desenvolvedor, você pode se unir a algumas centenas de outros desenvolvedores que contribuíram com o código Monero, via GitHub.
+
+O primeiro passo a se fazer quando se deseja começar um projeto com participação comunitária é entrar em contato com o grupo da Comunidade Monero, cujo foco é organizar todos os demais grupos de trabalho. Eles serão capazes de informar-lhe se a sua ideia já foi desenvolvida por alguém e com quais pessoas você deve trabalhar para aprimorá-la. No caso de a sua ideia ser completamente nova, eles disponibilizarão alguns recursos para auxiliá-lo, permitindo que sua ideia se torne realidade.
+
+Os principais desenvolvedores do Monero nos providenciaram inúmeras ferramentas para o sucesso dos grupos de trabalho, entre elas o Mattermost para comunicações e o Taiga para planejamento de projetos e gerenciamento de tarefas. As pessoas também podem participar de intensas discussões que ocorrem em vários canais de Slack e IRC, de grupos de conversa no Telegram, do Reddit via r/Monero ou dos subreddits dedicados a áreas específicas do Monero (como mineração e especulação) e perguntando ou respondendo perguntas sobre o Monero no Stack Exchange.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/teamwork/teamwork_pt-br.md]:
```suggestion
Os principais desenvolvedores do Monero nos providenciaram inúmeras ferramentas para o sucesso dos grupos de trabalho, entre elas o Mattermost para comunicações e o Taiga para planejamento de projetos e gerenciamento de tarefas. As pessoas também podem participar das discussões que ocorrem em vários canais de Slack e IRC, de grupos no Telegram, do Reddit via r/Monero ou dos subreddits dedicados a áreas específicas do Monero (como mineração e especulação) e perguntando ou respondendo perguntas sobre o Monero no Stack Exchange.
```

**Comment by lh1008** [comment]
Thanks @netrik182 and @rating89us for your review. @botoserelepe let us know when this would be ready to merge.

**Pull request merged**